bristen/packages.yaml

```
@@ -26,3 +27,24 @@ packages:
     externals:
     - spec: xpmem@2.9.6
       prefix: /usr
+
+  openmpi:
+    buildable: true
+    variants: +internal-pmix fabrics=cma,ofi,xpmem schedulers=slurm +cray-xpmem
+
+  libfabric:
+    buildable: true
+    externals:
+    - spec: libfabric@1.15.2.0 fabrics=cxi,tcp,udp,rxm,rxd
+      prefix: /opt/cray/libfabric/1.15.2.0/
+    version: ["git.e08d3161b8038ebfa50a4eb5bf76132e4eaeb137=main"]
```

I guess this could be bumped to 
```suggestion
    version: ["2.1.0"]
```
after https://github.com/spack/spack/pull/49549 is merged (that PR also adds the lnx provider).